packages/backend/src/index.ts

```diff
@@ -129,7 +129,7 @@ app.use(
             maxAge: (lightdashConfig.cookiesMaxAgeHours || 24) * 60 * 60 * 1000, // in ms
             secure: lightdashConfig.secureCookies,
             httpOnly: true,
-            sameSite: 'lax',
+            sameSite: 'strict',
```

Comment:
This change means now all embeds will be unauthenticated even if they are logged in on lightdash, right ?

Reply:
Since they have the same domain, cookies will still be sent.